### DISCUSSION TITLE
Changing background on chart hovering

### DISCUSSION THREAD

**Discussion by ocni-dtu** [mantinedev/mantine#6292] [Q&A] [answered]
Hi all,

Is it possible to customize the background of the hovered section in bar chart?

I have it completely black, which makes the tooltip hard to read. 
![image](https://github.com/mantinedev/mantine/assets/16247318/481afbf5-06a1-4344-a7a1-6f97bab3e697)

**Comment by rtivital** [comment]
I do not see such background on mantine.dev website

**Reply by ocni-dtu** [reply]
The above is what I get when I render the BarChart:

```
      <BarChart
        h={300}
        data={orderedData}
        dataKey='name'
        series={[{ name: 'value', color: '#4A5ABE' }]}
        yAxisLabel={t('result-table.count')}
        withTooltip={true}
        withBarValueLabel
        valueFormatter={(value) => new Intl.NumberFormat(i18n.resolvedLanguage).format(value)}
        tickLine='y'
      />
```
I'm using `@mantine/charts` 7.10.0, but also had the same behavior on 7.9.2

**Reply by rtivital** [reply]
Provide a sandbox with a minimal reproduction

**Reply by ocni-dtu** [reply]
Here you go: [CodeSandbox](https://codesandbox.io/p/sandbox/mantine-charts-sdg8qc?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clwqkloi10006286c1p0b9gno%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clwqkloi10002286cpvjisbrm%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clwqkloi10003286c0u1acyvt%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clwqkloi10005286c06a9v4e5%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clwqkloi10002286cpvjisbrm%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clwqkloi00001286c54ggl4hp%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.tsx%2522%257D%255D%252C%2522id%2522%253A%2522clwqkloi10002286cpvjisbrm%2522%252C%2522activeTabId%2522%253A%2522clwqkloi00001286c54ggl4hp%2522%257D%252C%2522clwqkloi10005286c06a9v4e5%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clwqkloi10004286c410heqv0%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clwqkloi10005286c06a9v4e5%2522%252C%2522activeTabId%2522%253A%2522clwqkloi10004286c410heqv0%2522%257D%252C%2522clwqkloi10003286c0u1acyvt%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clwqkloi10003286c0u1acyvt%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

Same issue:
![image](https://github.com/mantinedev/mantine/assets/16247318/0467d6f5-839d-4013-8586-702203aa7d3f)

**Reply by rtivital** [reply, marked as answer]
You did not import styles.
https://mantine.dev/charts/getting-started/#installation

**Reply by ocni-dtu** [reply]
🤦
